### PR TITLE
[stmt.if] Balance brackets and fix indentation.

### PR DESCRIPTION
**Pull request by Eelis**

### CODE REVIEW

File: source/statements.tex [hunk 1]
```diff
@@ -276,7 +276,7 @@
 template<typename T, typename ... Rest> void g(T&& p, Rest&& ...rs) {
   // ... handle \tcode{p}
 
-  if constexpr (sizeof...(rs) > 0)
+  if constexpr (sizeof...(rs) > 0) {
```
Comment (tkoeppe) [inline source/statements.tex]: This seems wrong. The closing brace matches the opening brace on line 276.

Comment (tkoeppe) [inline source/statements.tex]: No, this isn't right; the closing brace matches the opening brace on line 276. It's the indent of what follows that's wrong, and the brace on line 293 is extraneous.

**Comment by Eelis** [comment]
Ah ok, re-pushed

**Comment by tkoeppe** [comment]
Thanks. Can you please update the commit message?

**Comment by Eelis** [comment]
Done :)

**Comment by tkoeppe** [comment]
Can you please rebase, and also change the PR title to match the commit?

**Comment by Eelis** [comment]
Done

**Pull request merged**